Packs/CommonScripts/Scripts/FeedRelatedIndicatorsWidget/FeedRelatedIndicatorsWidget.py

```diff
@@ -2,33 +2,67 @@
 from CommonServerPython import *
 
 
+def create_related_indicator_object(value: str, type_: str, description: str):
+    """Creates a related indicator object to be added to table.
+
+    Args:
+        value (str): object value.
+        type_ (str): object type.
+        description (str): object description.
+
+    Returns:
+        Dict. Related indicator object.
+    """
+    indicator_object = {
+        'Value': value,
+        'Type': type_
+    }
+
+    descriptions = []
+
+    for desc in description.split(','):
```

Comment:
what will happen if there is only one description?
I think we can use argToList here.

Reply:
and add a test case for single and multiple descriptions.